app/queries/my_facilities/overview_query.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class MyFacilities::OverviewQuery
+  INACTIVITY_THRESHOLD_PERIOD = 1.week.ago
+  INACTIVITY_THRESHOLD_BPS = 10
+
+  def initialize(facilities = Facility.all)
+    @facilities = facilities
+  end
+
+  def inactive_facilities
+    facilities = @facilities.left_outer_joins(:blood_pressures)
+                            .where('blood_pressures.recorded_at IS NULL OR blood_pressures.recorded_at > ?',
+                                   INACTIVITY_THRESHOLD_PERIOD)
+                            .having('COUNT(blood_pressures) < ? ', INACTIVITY_THRESHOLD_BPS)
+                            .group('facilities.id')
+
+    Facility.where(id: facilities.pluck(:id))
```

Comment:
Can a view make this query easier? We don't have to address in this PR, but if so, let's add a story to tackle it later.

Reply:
We could create another view for computing `facility_activity_per_day`, but we'll do this as a separate PR. Adding a story for this.

Reply:
Neat, thanks. Go ahead and add a link to that story in this comment thread or PR description.

Reply:
https://www.pivotaltracker.com/story/show/170919252